src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/controller/AdjustmentsController.kt

```diff
@@ -130,6 +131,29 @@ class AdjustmentsController(
     }
   }
 
+  @PostMapping("/restore")
+  @Operation(
+    summary = "Restore a deleted adjustment",
+    description = "Restore a deleted adjustment",
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(responseCode = "200", description = "Adjustment restored"),
+      ApiResponse(responseCode = "401", description = "Unauthorised, requires a valid Oauth2 token"),
+      ApiResponse(responseCode = "404", description = "Adjustment not found"),
+    ],
+  )
+  @PreAuthorize("hasRole('ADJUSTMENTS_MAINTAINER')")
+  fun restore(
+    @Parameter(required = true, description = "The adjustment UUID")
+    @RequestBody
+    adjustments: RestoreAdjustmentsDto,
+  ) {
+    adjustmentsService.restore(adjustments).also {
+      eventService.create(adjustments.ids, it[0].person, AdjustmentSource.DPS)
```

Comment:
is there a scenario where the ids submitted span across multiple people? or is it always the a single person and multiple ids supplied?

If the former then just getting the first person would result in an event with adjustment ids which don't belong to the person, if the latter then this is good